test/beast/core/test_buffer.hpp

```diff
@@ -360,21 +360,21 @@ test_mutable_dynamic_buffer(
 
 /** Test an instance of a dynamic buffer or mutable dynamic buffer.
```

Comment:
There should be no such thing as "mutable dynamic buffer". All of Beast's dynamic buffers should support both `const` and non-`const` `data()`. You might need a separate commit for that.

Reply:
I don't understand what you mean.
The code comment about was not written by me.

Reply:
Yep I know, lets discuss it